### DISCUSSION TITLE
NOOB Questions Home Assistant integration

### DISCUSSION THREAD

**Discussion by McPaHa** [teslamate-org/teslamate#4083] [Q&A]
Hi!

Stupid questions, sorry. 

When trying to integrate Teslamate to HA - what are:

username: !secret tesla_username
password: !secret tesla_password

So where to get tesla_username and password?

Also... What should be here:

device: &teslamate_device_info
-->      configuration_url: https://teslamate.zxxz.io/

if Teslamate is running same Debian PC than HA with docker.

**Comment by greggitter** [comment]
I have it in my config but it's commented out.  But all the data from teslamate is basically via mqtt so there's no connection directly from HA to tesla cloud.  However, reading at the top (link below), for those that want to enable commands from HA or TM to the car it looks like that might be required.  I don't enable commands...my setup is basically read-only.  Anyhow that's how I read it...maybe someone can clarify if my assumption is wrong.

Ref: https://docs.teslamate.org/docs/integrations/home_assistant
